src/data/option/defs.lean

```diff
@@ -104,6 +104,24 @@ inductive rel (r : α → β → Prop) : option α → option β → Prop
 | some {a b} : r a b → rel (some a) (some b)
 | none       : rel none none
 
+/-- Partial bind. If for some `x : option α`, `f : Π (a : α), a ∈ x → option β` is a
+  partial function defined on `a : α` satisfying `p` giving an `option β`,
+  then `pbind x f h` is essentially the same as `bind x f`
+  but is defined only when all `x = some a` and the `a` satisfy `p`, using the proof
+  to apply `f`. -/
+@[simp] def pbind : Π (x : option α), (Π (a : α), a ∈ x → option β) → option β
+| none     _ := none
+| (some a) f := f a rfl
```

Comment:
I don't see `p` anywhere in the code - is this comment wrong? Or should `a ∈ x` be `p x`?

Reply:
I tried to reword the docstring.